pkg/kubelet/status/status_manager.go

```diff
@@ -19,6 +19,8 @@ package status
 import (
 	"context"
 	"fmt"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
```

Comment:
These need to be in the appropriate section of imports.

Reply:
Addressed, @smarterclayton